src/pages/experiences.jsx

```diff
@@ -0,0 +1,582 @@
+              >
+                Infura
+              </a>
+            </div>
+            <div className="ml-4">
+              Join the Decentralized Infrastructure Network (DIN) and Consensys
+              Diligence team for a Smart Contract Capture the Flag (CTF)
+              Farcaster Frames challenge to solve riddles and mint a limited
+              edition Goerli Commemorative NFT. Celebrating Goerli's launch at
+              Göorlicon in Feb 2019, this puzzle will be deployed at ETHBerlin
+              and available to the broader audience.
+            </div>
+          </div>
+
+          <div className="mt-8">
+            <div className="inline-flex flex-wrap items-center">
+              <h3 className="font-bold text-lg mr-1">Teledisko</h3>
+              <span>by</span>
+              <a
+                href="https://www.teledisko.com/"
+                target="blank"
+                rel="noreferrer noopener"
+                className="underline ml-1"
+              >
+                TelediskoDAO
+              </a>
+              <LocationButton loc={locations.yard1} />
+            </div>
+            <div className="ml-4">
+              The teledisko needs no further explanation. Brought to you by
+              telediskoDAO, it’s the smallest disco on earth - literally an
+              upcycled telephone booth with a disco in it.
+            </div>
+          </div>
+
+          <div className="mt-8">
+            <div className="inline-flex flex-wrap items-center">
+              <h3 className="font-bold text-lg mr-1">Donut Wall</h3>
+              <span>by</span>
+              <a
+                href="https://missing-link.io/"
+                target="blank"
+                rel="noreferrer noopener"
+                className="underline ml-1"
+              >
+                Missing Link
+              </a>
+              <LocationButton loc={locations.yard0} />
+            </div>
+            <div className="ml-4">
+              Back by popular demand is the donut wall! It’s exactly what it
+              says on the tin - A wall of donuts for you to get your sugar hit
+              during the weekend.
+            </div>
+          </div>
+
+          <div className="mt-8">
+            <div className="inline-flex flex-wrap items-center">
+              <h3 className="font-bold text-lg mr-1">After party</h3>
+              <span>by</span>
+              <a
+                href="https://entropy.xyz/"
+                target="blank"
+                rel="noreferrer noopener"
+                className="underline ml-1"
+              >
+                Entropy
+              </a>{" "}
+              &{" "}
+              <a
+                href="https://fuel.network/"
+                target="blank"
+                rel="noreferrer noopener"
+                className="underline ml-1"
+              >
+                Fuel
+              </a>
+              <LocationButton loc={locations.aeden} />
```

Comment:
Shall we make this a link instead of a button since it moves you to an external resource?